github/Repository.py

```diff
@@ -1518,6 +1518,7 @@ def get_branch(self, branch):
         headers, data = self._requester.requestJsonAndCheck(
             "GET", self.url + "/branches/" + branch
         )
+        data['is_organization'] = self.organization is not None
```

Comment:
This change causes Repository object to be loaded in tests, negating lazy load https://github.com/PyGithub/PyGithub/blob/dd158556a838a180d966925c824100f320a5fcfc/tests/Branch.py#L45

I'm not sure if the test data needs rerecording or if there is a different fix for the resulting failed tests.

Reply:
Yeah, I think you might want to duck under the covers there and use the underscored attribute.

Reply:
I did look at doing that, but as I understand the lazy loading, the organization key would not exist before completing the object. The result would be a lazy loaded Repository would always look like a user repo.